### DISCUSSION TITLE
I’m banned from uYouPlus

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#21] [General]
Unfortunately I’m just dumb as it was. I was banned for a reason and I guess I just have to deal with it. I will still improve this. I’m sorry to PoomSmart, the thing I said about you taking a break was some stupid thing I came up with, I was just trying to not be mean or anything, having this ban will really suck because I was technically one of the people who have gave uYou+ some more popularity by promoting it across socials. And making forks that helped its popularity, I’ve been with the uYou+ community for a year and I technically did my best helping it out with some features but the sad thing is I’m now unable to do any pull requests now so it really sucks to see that disabled for me. I know I’m moaning about that ban but it really shows that I really can’t do anything but glad that y’all are sticking together and hope uYou+ still continues but to the devs who are reading this, very sorry for my stupidest and dumbest message I’ve made. I really didn’t expect those turn of events.

**Comment by bobkinect98** [comment]
oh. well that's a downer.
im not really in the community for uYou+, so i don't really know whats going on, but glad that your owning up to whatever happened behind the scenes.
btw, is there a duration on this ban or its permanent one?

**Reply by arichornlover** [reply]
I think it’s a pernament one. It sucks but it was my fault for telling a dev to take a break. but it’s gonna be unfortunate to not make pull requests, issues or discussions there 🤦

**Reply by bobkinect98** [reply]
ah alright, is there any good alternatives to uYouEnhanced? since its not being updated no longer, might as well ask.

**Reply by arichornlover** [reply]
Nah I’m still updating it uYouEnhanced but there are alternatives aswell thanks for asking.
https://github.com/Balackburn/YTLitePlus - YTLitePlus tweak made by Balackburn and YTLite tweak by Dayanch96
https://github.com/arichornlover/YouTubeRebornPlus - YouTubeRebornPlus is a tweak that was based off of qnblackcat’s CercubePlus tweak that includes a forked version of YouTube Reborn and includes additional features that can potentially improve LillieH1000’s YouTube Reborn tweak.

**Comment by arichornlover** [comment]
> Unfortunately I’m just dumb as it was. I was banned for a reason and I guess I just have to deal with it.

but honestly I did the right decision as well. You know some things in life people can get banned from stuff and the stuff they contribute to and it does suck because that somebody like me supports can still support the tweak after getting banned. But hey what I did was right and qnblackcat. what I said about PoomSmart was also important. Telling the dev to take a break from YouTube Tweaks (including uYou+) would actually help you out. and after you returned to YouTube Tweaks, you can actually feel better and get more ideas for YouTube-related tweaks. I know I’m wrong but you gotta understand what makes you, you with wonderful creations. I take breaks too and I was able to do Improvements with uYou+ by adding a ton of unique features to the tweak made by me. Look I know most of y’all aren’t gonna read this but just so you know, I’m sorry.

**Reply by arichornlover** [reply]
It’s getting annoying at this point. qnblackcat please stop telling people that I don’t respect you. Whatever you got that from has to be a lie because I respected y’all when uYouPlus was a thing and ever since I was banned I improved on how to make stuff with Theos. Heck I even improved upon my tweak LowContrastMode to support the latest version and then made a whole new tweak that spoofs the app version in the YouTube Settings to change the features/api/ui in the app. Even made a tweak that hides the pivot bar indicator for the apps. YouTube, TikTok, Twitter/X & Instagram. and qnblackcat I am not even ruining your reputation or disrespecting you. You are in the README.md and in some releases because you are the the reason for the existence of the tweak uYouEnhanced.
